server/i18nlint/i18nlint.js

```diff
@@ -9,7 +9,7 @@ var fs = require( 'fs' ),
 	babel = require( 'babel-core' ),
 	preProcessXGettextJSMatch = require( '../i18n/preprocess-xgettextjs-match.js' ),
 	SourceMapConsumer = require( 'source-map' ).SourceMapConsumer,
-	tokenize = require( '../../client/lib/interpolate-components/tokenize.js' ),
```

Comment:
Do we need to depend on the folder structure of the `npm` module here? Can we expose `tokenize` either as part of the main `interpolate-components` export (`interpolateComponents.tokenize()`) or as an alternate export (`import { tokenize } from 'interpolate-components'`).